### PR TITLE
P20-1180: Add Global Edition region switch confirm

### PR DESCRIPTION
**Pull request by artem-vavilov**
## Demo
global-edition-region-switch-confirm

## Links
- JIRA task: [P20-1180](https://codedotorg.atlassian.net/browse/P20-1180)
- Related PR: https://github.com/code-dot-org/code-dot-org/pull/62061

### CODE REVIEW

File: lib/cdo/rack/global_edition.rb [hunk 1]
```diff
@@ -10,6 +10,9 @@ class GlobalEdition
     AVAILABLE_REGIONS = Set[
       FARSI_REGION = 'fa',
     ].freeze
+    COUNTIES_REGIONS = {
```
Comment (wilkie) [inline lib/cdo/rack/global_edition.rb]:
```suggestion
    COUNTRIES_REGIONS = {
```

File: dashboard/test/ui/utils/selenium_browser.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 require 'selenium/webdriver'
-require 'webdrivers'
+# require 'webdrivers'
```
Comment (wilkie) [inline dashboard/test/ui/utils/selenium_browser.rb]:
```suggestion
require 'webdrivers'
```

File: dashboard/app/helpers/global_edition_helper.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module GlobalEditionHelper
+  def global_edition_region_switch_confirm_partial
+    return unless Rack::GlobalEdition::TARGET_HOSTNAMES.include?(request.hostname)
+
+    country_code = request.country_code
+    ge_region = Rack::GlobalEdition::COUNTIES_REGIONS[country_code]
```
Comment (wilkie) [inline dashboard/app/helpers/global_edition_helper.rb]:
```suggestion
    ge_region = Rack::GlobalEdition::COUNTRIES_REGIONS[country_code]
```

File: lib/cdo/rack/global_edition.rb (hunk 1 above)
Comment (artem-vavilov) [reply lib/cdo/rack/global_edition.rb]: 🤦🏻

**Review by wilkie — APPROVED** [review]

**Pull request merged**